### DISCUSSION TITLE
Android设备低内存时播放不了

### DISCUSSION THREAD

**Discussion by GeekBugs** [Tencent/libpag#2559] [Q&A] [answered]
Android设备在可运行内存低（不到1G）的情况下，是不是播放不出来？

**Comment by kevingpqi123** [comment, marked as answer]
没有这个限制，可以跑下 pag 的官方 demo，有一种情况会无法播放，如果在开发者选项中关闭了动画时长伸缩，会影响到系统的 ValueAnimator， pag 的动画播放依赖 ValueAnimator，此时是无法播放的
